components/basic.css

```diff
@@ -12,7 +12,7 @@ header {
 h1 {
   text-align: center;
   font-family: var(--title);
-  color:var(--first-color);
+  color: var(--first-color);
```

Comment:
really cool that you used this fuction with var, will also try to implement it :-D